client_secrets.json.default

```diff
@@ -1,12 +1,12 @@
 {
     "web": {
-        "auth_uri": "https://keycloak-dev.cirg.washington.edu/auth/realms/Stayhome/protocol/openid-connect/auth",
-        "client_id": "stayhome-population-app",
-        "client_secret": "ask-paul",
-    	"issuer": "https://keycloak-dev.cirg.washington.edu/auth/realms/Stayhome",
-        "redirect_uris": ["http://localhost:5000/oidc_callback"],
-        "userinfo_uri": "https://keycloak-dev.cirg.washington.edu/auth/realms/Stayhome/protocol/openid-connect/userinfo",
-        "token_uri": "https://keycloak-dev.cirg.washington.edu/auth/realms/Stayhome/protocol/openid-connect/token",
-        "token_introspection_uri": "https://keycloak-dev.cirg.washington.edu/auth/realms/Stayhome/protocol/openid-connect/token/introspect"
+        "auth_uri": "https://keycloak-dev.cirg.washington.edu/auth/realms/cosri-launcher/protocol/openid-connect/auth",
+        "client_id": "cosri-patientsearch",
+        "client_secret": "ask-paul or check keycloak-dev",
+    	"issuer": "https://keycloak-dev.cirg.washington.edu/auth/realms/cosri-launcher",
+        "redirect_uris": ["http://localhost:8000/oidc_callback"],
+        "userinfo_uri": "https://keycloak-dev.cirg.washington.edu/auth/realms/cosri-launcher/protocol/openid-connect/userinfo",
+        "token_uri": "https://keycloak-dev.cirg.washington.edu/auth/realms/cosri-launcher/protocol/openid-connect/token",
+        "token_introspection_uri": "https://keycloak-dev.cirg.washington.edu/auth/realms/cosri-launcher/protocol/openid-connect/token/introspect"
```

Comment:
Would be better to read these from env vars for consistent configuration

Reply:
i agree - but request we make that a TODO, as the flask-oidc lib doesn't document how to easily move out of this magic filename.